Description: Pass DPI, along with format, in plot format configuration
Forwarded: yes
Author: Sandro Tosi <morph@debian.org>

diff -urNad matplotlib~/doc/make.py matplotlib/doc/make.py
--- matplotlib~/doc/make.py	2009-08-01 21:14:12.000000000 +0200
+++ matplotlib/doc/make.py	2009-10-09 08:56:04.000000000 +0200
@@ -30,7 +30,7 @@
     check_build()
     shutil.copy('../lib/matplotlib/mpl-data/matplotlibrc', '_static/matplotlibrc')
     if small_docs:
-        options = "-D plot_formats=\"['png']\""
+        options = "-D plot_formats=\"[('png', 80)]\""
     else:
         options = ''
     if os.system('sphinx-build %s -P -b html -d build/doctrees . build/html' % options):
